wire-ios-data-model/Source/MLS/MLSGroupID.swift

```diff
@@ -24,60 +24,34 @@ public struct MLSGroupID: Equatable, Hashable {
 
     // MARK: - Properties
 
-    let data: Data
+    public let data: Data
 
     // MARK: - Life cycle
 
     public init?(base64Encoded string: String) {
-        guard let data = Data(base64Encoded: string) else { return nil }
+        guard !string.isEmpty, let data = Data(base64Encoded: string) else { return nil }
```

Comment:
@David-Henner I added this empty-string check because it was there in the duplicate method in `MLSEventProcessor.swift`. However, an empty string correctly base64-decodes into empty data and the main initialiser does allow empty data. This is somewhat inconsistent in my opinion.

Reply:
Agreed. Not sure what I think of it. I'd like to avoid having an optional main initialiser